src/ORM/Connect/Database.php

```diff
@@ -229,6 +248,30 @@ protected function benchmarkQuery($sql, $callback, $parameters = array())
         }
     }
 
+    /**
+     * Display query message
+     *
+     * @param int $queryCount
+     * @param mixed $query
+     * @param float $endtime
+     */
+    private function displayQuery($queryCount, $query, $endtime)
```

Comment:
could use `$this->queryCount` rather than having it as an explicit method argument. That would reduce number of arguments and improve readability a little bit